src/LinkedIn.php

```diff
@@ -277,7 +277,7 @@ protected function getUserFromAccessToken()
         try {
             return $this->api('GET', '/v1/people/~:(id,firstName,lastName,headline)');
         } catch (LinkedInApiException $e) {
-            return;
+            return null;
```

Comment:
`return null` is redundant, you should use ´return´ only. This is recommended by sensiolabs insight if I remember correctly.